### PR TITLE
Add comment replies

### LINKED ISSUE #4389: Order code comments by review
It would be nice if code comments will be ordered by Review > Date. This will have the side-effect that we can show the complete conversation in the "conversations" tab. At the moment only the first comment of line gets shown at reviews in conversation tab.

My idea would be to introduce a new column at `comment` called `reply_to` containing the review id of the reply.

Blocked by #3748

### PR DESCRIPTION
**Pull request by jonasfranz**
Partially fixes #4389 and #5100.

This PR makes it possible to reply to review comments. These replies will be also shown at the "Conversation" tab.

At the moment it is only possible to reply to comments if there is already a review at the specific line. That can be fixed, but requires translations. We should discuss about including this in 1.6.

Without this enhancement the review comment feature seems buggy to the end user.

### CODE REVIEW

File: routers/repo/pull_review.go [hunk 1]
```diff
@@ -63,6 +63,10 @@ func CreateCodeComment(ctx *context.Context, form auth.CodeCommentForm) {
 			}
 		}
 	}
+	reviewID := review.ID
```
Comment (kolaente) [inline routers/repo/pull_review.go]: Why are you putting this in an extra variable instead of directly modifiying `review.ID`?

**Review by techknowlogick — APPROVED** [review]
Same nit as @kolaente, but otherwise LGTM.

**Comment by jonasfranz** [comment]
@kolaente @techknowlogick done

**Review by appleboy — APPROVED** [review]

**Review by kolaente — APPROVED** [review]

**Comment by jonasfranz** [comment]
@go-gitea/owners Drone isn't reporting a status

**Comment by filipnavara** [comment]
I submitted it when GitHub was down and the PR page was not accessible for couple of hours, or accessible only from some regions.

**Comment by filipnavara** [comment]
GitHub is still working through the backlogs (https://status.github.com/), but it should mostly work now.

**Comment by techknowlogick** [comment]
@JonasFranzDEV please send backport.

**Pull request merged**